packages/gcf-utils/package.json

```
@@ -41,7 +41,7 @@
     "express": "^4.18.1",
     "gaxios": "^5.0.1",
     "get-stream": "^6.0.1",
-    "google-gax": "^3.3.1",
+    "google-gax": "^3.5.2"
```

looks like we're missing a trailing comma
```suggestion
    "google-gax": "^3.5.2",
```